ipfs-8-to-9/migration/go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/ipfs/fs-repo-migrations/ipfs-8-to-9/migration
+
+go 1.13
+
+require (
+	github.com/hsanjuan/go-libp2p-http v0.0.2 // indirect
```

Comment:
This should be libp2p/go-libp2p-http. Need to depend on ipfs-master-fake-tagged version.